static/blog-seo/config.json

```diff
@@ -715,6 +715,7 @@
     {
         "path": "/blog/totp-why-you-need-it-and-how-it-works",
         "metaTags": [
+            "<meta name=\"author\" content=\"Joel Coutinho\" /> ",
```

Comment:
since joel is on multiple posts now, add this on all the posts he is a currently an author in.

Reply:
done